README.md

```diff
@@ -4,8 +4,8 @@
 
 # Storybook
 
-* [Production](https://ucsb-cs156-w22.github.io/team04-w22-6pm-HappyCows-docs/)
-* [QA](https://ucsb-cs156-w22.github.io/team04-w22-6pm-HappyCows-docs-qa/)
+* [Production](https://github.com/ucsb-cs156-s22/s22-5pm-happycows-docs)
+* [QA](https://github.com/ucsb-cs156-s22/s22-5pm-happycows-docs-qa)
```

Comment:
Can we get the links to the actual deployed Storybooks here instead of the link to the repo?

Reply:
just fixed!